pkg/sources/adapter/webhooksource/webhook.go

```diff
@@ -98,6 +97,10 @@ func runHandler(ctx context.Context, s *http.Server) error {
 // handleAll receives all webhook events at a single resource, it
 // is up to this function to parse event wrapper and dispatch.
 func (h *webhookHandler) handleAll(w http.ResponseWriter, r *http.Request) {
+	if h.corsAllowOrigin != "" {
```

Comment:
@cab105 is this right? Or should this be checking aginst a `nil`?

Reply:
Much better, thank you!

Reply:
okay! thank you